streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImpl.java

```diff
@@ -247,6 +255,7 @@ private void determineIsQueryable(final String queryableStoreName) {
         );
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Do we need this one? `count(StateStoreSupplier)` already has a `SuppressWarnings` so i think this one isn't needed? Same for `reduce` etc

Reply:
We do need... Same below -- I guess Intellij default are different that gradle build warnings.